package.json

```diff
@@ -6,6 +6,12 @@
     "repository": "https://github.com/AndelaOSP/wire.git",
     "author": "Mbarak Mbigo <mbarak.mbigo@andela.com>",
     "license": "MIT",
+    "jest": {
```

Comment:
I think it would have been more organized if we added this code to a config  file  as compared to adding it in the `package.json` file. Let's have only `scripts` and `dependencies` in this file.

Reply:
@Mbarak-Mbigo  according to the jest documentation we can have the configurations defined in the package.json file of your project. and since the configurations are just two lines of code and it is still on the level as the scripts why not have them in the package.json instead of having too many files.. @Mnickii  @Mercy-Muchai  what do u guys think.

Reply:
I think we should move the configurations to a file for the purpose of consistency and organization. Just like we separated common `webpack configurations, dev` and `production configs`. We also could have put `babel configurations to webpack files` but we are not doing that for organisational purposes and consistency`.

Reply:
Well, if all config details are in separate config files for the above purposes we can do it that way. I guess then we would need to have a `jest.config.js` @RayNjeri

Reply:
Based on the information you have provided @RayNjeri on the `config bug issue`, I recommend we merge it with this configuration for now and once we get a workaround or a viable solution we transfer the configurations to their respective file.